# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/drbd7/tail-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./drbd/Makefile.orig	2005-08-24 10:45:39.000000000 +0200
+++ ./drbd/Makefile	2005-08-24 10:44:40.000000000 +0200
@@ -109,7 +109,7 @@
 	@mv .kernelrelease.new .kernelrelease
 	@echo -n "Memorizing module configuration ... "
 	@{ echo -e "#\n# drbd.o was compiled with"          ; \
-	  echo "#  `gcc -v 2>&1 | tail -1`"                 ; \
+	  echo "#  `gcc -v 2>&1 | tail -n -1`"              ; \
 	  echo "# against this kernelrelease:"              ; \
 	  sed 's/^/#  /' .kernelrelease                     ; \
           echo    "# kernel .config from"                   ; \
